lib/OpenQA/Script/CloneJob.pm

```diff
@@ -238,8 +238,11 @@ sub clone_job ($jobid, $options, $clone_map = {}, $depth = 0, $parent_jobid = 0)
 
     clone_job_download_assets($jobid, $job, $remote, $remote_url, $ua, $options) unless $options->{'skip-download'};
 
-    my $url      = $local_url->clone;
+    my $url        = $local_url->clone;
+    my $source_url = $remote_url->clone;
+    $source_url = $source_url->path('/tests/' . "$jobid");
     my %settings = %{$job->{settings}};
+    $settings{CLONED_FROM} = "$source_url";
```

Comment:
This is not bash:
```suggestion
    $settings{CLONED_FROM} = $source_url;
```

Reply:
I agree, perl isnt bash .. but this is https://docs.mojolicious.org/Mojo/Path#to_string ( if I corectly understand docu )